tools/pot/openvino/tools/pot/graph/graph_utils.py

```diff
@@ -9,7 +9,10 @@
 from openvino.tools.mo.utils.logger import init_logger
 from openvino.runtime import Core  # pylint: disable=E0401,E0611
 from openvino.runtime.passes import Manager # pylint: disable=E0401,E0611
-from openvino.offline_transformations_pybind import apply_pot_transformations # pylint: disable=import-error,no-name-in-module
+try:
+    from openvino.offline_transformations import apply_pot_transformations # pylint: disable=import-error,no-name-in-module
+except ImportError:
+    from openvino.offline_transformations_pybind import apply_pot_transformations # pylint: disable=import-error,no-name-in-module
```

Comment:
why do we need it? where POT can take `offline_transformations_pybind `?

Reply:
It was added to pass POT CI -- it's a separate action to switch their CI for using OpenVINO from this branch,
This addition was agreed with POT team (@KodiaqQ) and later will be removed by him.